server/views/pages/personSearchInPpudResults.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "../partials/layout.njk" %}
+
```

Comment:
This page will need to move into the recommendations dir when we wire it correctly.